mempool/clist_mempool.go

```diff
@@ -130,6 +149,16 @@ func (mem *CListMempool) EnableTxsAvailable() {
 	mem.txsAvailable = make(chan struct{}, 1)
 }
 
+func (mem *CListMempool) SetTxRemovedCallback(cb func(txKey types.TxKey)) {
+	mem.removeTxOnReactor = cb
+}
```

Comment:
check it is called once? (defensive programming)

Reply:
Not sure if it's needed. This function is only called in the reactor constructor.